lib/percy/client/environment.rb

```diff
@@ -129,8 +132,8 @@ def self.branch
         end
 
         if result == ''
-          STDERR.puts '[percy] Warning: not in a git repo, setting PERCY_BRANCH to "master".'
-          result = 'master'
+          STDERR.puts '[percy] Warning: not in a git repo, no BRANCH detected.'
```

Comment:
`BRANCH` is not an env var we support -- maybe just lowercase it to make this clearer?